examples/chef/common/clusters/media-input/MediaInputManager.cpp

```diff
@@ -56,11 +67,22 @@ uint8_t MediaInputManager::HandleGetCurrentInput()
 
 bool MediaInputManager::HandleSelectInput(const uint8_t index)
 {
+    if (mCurrentInput == index)
+    {
+        ChipLogProgress(Zcl, "CurrentInput is same as new value: %u", index);
+        return true;
+    }
     for (auto const & inputData : mInputs)
     {
         if (inputData.index == index)
         {
             mCurrentInput = index;
+            // Sync the CurrentInput to attribute storage while reporting changes
```

Comment:
So what I don't get is still: why are we storing the value in multiple places in memory and then having to sync?

Reply:
Let's remove `mCurrentInput` management and only use Attributes::CurrentInput::Set/Get

Reply:
@bzbarsky-apple @tcarmelveilleux  `mCurrentInput` is removed. PTAL